packages/KModal/KModal.vue

```diff
@@ -5,39 +5,43 @@
     class="k-modal"
     role="dialog">
     <div
-      class="modal-backdrop"
+      class="k-modal-backdrop modal-backdrop"
       @click="close">
 
       <div
-        class="modal-dialog"
+        class="k-modal-dialog modal-dialog"
         @click.stop>
-        <div class="modal-content">
+        <div class="k-modal-content modal-content">
           <div
```

Comment:
This should be a heading tag.

Reply:
I don't really want to mess with styles associated with headings. Is there a `role` I can use?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role will work?